xline/src/storage/index.rs

```diff
@@ -91,36 +131,42 @@ pub(crate) trait IndexOperate {
 
 impl IndexOperate for Index {
     fn get(&self, key: &[u8], range_end: &[u8], revision: i64) -> Vec<Revision> {
-        let index = self.index.lock();
+        let inner = self.inner.lock();
         match RangeType::get_range_type(key, range_end) {
-            RangeType::OneKey => index
+            RangeType::OneKey => inner
+                .index
                 .get(key)
                 .and_then(|revs| Self::get_revision(revs, revision))
                 .map(|rev| vec![rev])
                 .unwrap_or_default(),
-            RangeType::AllKeys => index
+            RangeType::AllKeys => inner
+                .index
                 .values()
                 .filter_map(|revs| Self::get_revision(revs, revision))
                 .collect(),
-            RangeType::Range => index
+            RangeType::Range => inner
+                .index
                 .range(KeyRange::new(key, range_end))
                 .filter_map(|(_k, revs)| Self::get_revision(revs, revision))
                 .collect(),
         }
     }
 
     fn get_from_rev(&self, key: &[u8], range_end: &[u8], revision: i64) -> Vec<Revision> {
-        let index = self.index.lock();
+        let inner = self.inner.lock();
         match RangeType::get_range_type(key, range_end) {
-            RangeType::OneKey => index
+            RangeType::OneKey => inner
+                .index
                 .get(key)
                 .map(|revs| Self::filter_revision(revs, revision))
                 .unwrap_or_default(),
-            RangeType::AllKeys => index
+            RangeType::AllKeys => inner
+                .index
```

Comment:
The test case `test_get` only covers the `RangeType::OneKey` match arm, please enrich this test case to cover all match arms.

Reply:
Fixed